envs/feedstock-patches/boost_cpp/build-only-required.patch

```diff
@@ -20,6 +29,24 @@ index 6f6b558..2ea54f2 100644
    patches:
      # https://svn.boost.org/trac10/attachment/ticket/11120/python_jam.patch
      - python_jam.patch
+@@ -29,7 +29,7 @@ requirements:
+     - {{ compiler('c') }}
+     - {{ compiler('cxx') }}
+   host:
+-    - icu                   # [unix]
++    - icu 73.1                  # [unix]
+     - bzip2                 # [unix]
+     - zlib
+ 
+@@ -42,7 +42,7 @@ outputs:
+         - {{ compiler('c') }}
+         - {{ compiler('cxx') }}
+       host:
+-        - icu               # [unix]
++        - icu 73.1              # [unix]
```

Comment:
for all platforms, or is this specific to s390x ?

Reply:
I missed that Alex, but now I have added a filter for s390x.